client/scripts/app.ts

```diff
@@ -648,6 +659,25 @@ Promise.all([$.ready, $.get('/api/domain')]).then(
               scoped: false,
               deferChain: true,
             }),
+            '/dashboard/': importRoute('views/pages/user_dashboard', {
```

Comment:
We don't need any of these route available on custom-domains. They should do nothing? Or redirect to `views/pages/discussions` as we're "scoped inside the community" at this point in the app.

Reply:
Just pushed an improved routes commit that handles custom domains better + has a more elegant system for Activity type-identification